modules/client/examples/05-encoders-decoders/12-apply-uninstallation.ts

```diff
@@ -0,0 +1,109 @@
+/* MARKDOWN
+---
+title: Apply Uninstallation
+---
+
+## Apply a Plugin Uninstallation
+
+Encodes the action of applying a plugin uninstallation.
+
+### Encoding
+*/
+
+import {
+  ApplyUninstallationParams,
+  Client,
+  DaoAction,
+  DecodedApplyUninstallationParams,
+  PermissionOperationType,
+} from "@aragon/sdk-client";
+import { hexToBytes } from "@aragon/sdk-common";
+import { context } from "../index";
+import { PermissionIds } from "../../dist/interfaces";
+
+// Instantiates an Aragon OSx SDK client.
+const client: Client = new Client(context);
+
+// This variable contains the values received on the prepareUninstallation() method
+const applyUninstallationParams: ApplyUninstallationParams = {
+  permissions: [{
+    operation: PermissionOperationType.REVOKE,
+    permissionId: hexToBytes(PermissionIds.EXECUTE_PERMISSION_ID),
```

Comment:
can't we accept it as a hex string? This is a fixed length value with no data processing applied

Reply:
Yes, the contracts accept `BytesLike` which is basically `Uint8Array | number[] | string`, so we can accept the string for this if you want

Reply:
ok, but I mean about the only-one-correct-way of receiving data on the SDK side (the internal Ethers behaviour should be independent)

Reply:
We are generally accepoting `Uint8Array` for all this kind of hex data, we can change to string in this case or Uint8Array | string